MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -183,26 +184,24 @@
    * @param customerPlayerData basic playback data set by the Application.
    * @param customerVideoData basic Video data set by the Application.
    * @param customerViewData basic View data set by the application.
-   * @param sentryEnabled if set to true the underlying {@link MuxStats} will report internal errors
-   *                      to the backend.
    * @param networkRequest internet interface implementation.
    */
   @Deprecated
```

Comment:
I think we used to document this, or we've at least given it out to customers. I'm fine removing this as an option completely, but I think we'd need to technically major version the release.

Reply:
I agree, we should keep the constructor signature intact and ignore the parameter.